src/ui/shared/AddOrEditFormFooter.tsx

```diff
@@ -2,21 +2,40 @@ import { Button, Flex } from 'fds/components';
 import * as React from 'react';
 
 import ErrorToast from 'fontoxml-feedback/src/ErrorToast';
-import { RecoveryOption } from 'fontoxml-feedback/src/types';
+import { CardContentComponentProps, RecoveryOption } from 'fontoxml-feedback/src/types';
+
 import t from 'fontoxml-localization/src/t';
 
 import ResponsiveButtonSpacer from './ResponsiveButtonSpacer';
+import { AnnotationErrorType } from 'fontoxml-review-reference-configuration/src/types';
 
-function determineSaveButtonLabel(error, isLoading) {
+function determineSaveButtonLabel(
+	error: AnnotationErrorType,
+	isLoading: boolean
+) {
 	if (isLoading) {
 		return t('Saving…');
 	}
 
-	return error && error.recovery === RecoveryOption.RETRYABLE
+	return typeof error !== 'number' && 
+		error &&
+		error.recovery === RecoveryOption.RETRYABLE
 		? t('Retry save')
 		: t('Save');
 }
 
+type Props = {
+	error:  AnnotationErrorType;
+	isDisabled: boolean;
+	isLoading: boolean;
+	isSubmitDisabled: boolean;
+	onCancel: CardContentComponentProps['onReviewAnnotationFormCancel'];
+	onReviewAnnotationRefresh: CardContentComponentProps['onReviewAnnotationRefresh'];
+	onSubmit: CardContentComponentProps['onReviewAnnotationFormSubmit'];
+};
+
+const paddingSize = { top: 'l' };
```

Comment:
To fix the TS error from line 58, we need to indicate that this is a `PaddingSize` object.

```
const paddingSize: PaddingSize = { top: 'l' };
```